website/docs/language/settings/index.mdx

```diff
@@ -1,61 +1,61 @@
 ---
 description: >-
-  The opentf block allows you to configure OpenTF behavior, including the
-  OpenTF version, backend, integration with TACOS (TF Automation and Collaboration Software),
+  The tofu block allows you to configure OpenTofu behavior, including the
+  OpenTofu version, backend, integration with TACOS (TF Automation and Collaboration Software),
   and required providers.
 ---
 
-# OpenTF Settings
+# OpenTofu Settings
 
-The special `opentf` configuration block type is used to configure some
-behaviors of OpenTF itself, such as requiring a minimum OpenTF version to
+The special `tofu` configuration block type is used to configure some
+behaviors of OpenTofu itself, such as requiring a minimum OpenTofu version to
 apply your configuration.
 
-## OpenTF Block Syntax
+## OpenTofu Block Syntax
 
-OpenTF settings are gathered together into `opentf` blocks:
+OpenTofu settings are gathered together into `tofu` blocks:
 
 ```hcl
 terraform {
   # ...
 }
 ```
 
-Each `opentf` block can contain a number of settings related to OpenTF's
-behavior. Within a `opentf` block, only constant values can be used;
+Each `tofu` block can contain a number of settings related to OpenTofu's
```

Comment:
This might be mistake from previous renaming, I think it should be `terraform` block

Reply:
Correct, I'll add this to the big issue im going to make :D